### PR TITLE
Complete the alert

### PR DESCRIPTION
**Pull request by KBhautik6**
### alerts
Screenshot 2023-12-26 at 7 37 19 PM

### outline alert

Screenshot 2023-12-26 at 7 37 26 PM

- [x] Checking the code is duplicate or not?
- [x] Is these code has bug 
- [ ] Is code is check by shubhum sir or ankit sir or tara ma'am?
- [ ] Are you going to merge this or not?

### CODE REVIEW

**Comment by shubham-spreadd** [comment]
@KBhautik6 please check component alignments

File: src/stories/Alert.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  isoutline,
+  border,
+  ishover,
+}) => (
+  <Alert
+    type={type}
+    variant={variant}
+    isoutline={isoutline}
+    border={border}
+    ishover={ishover}
```
Comment (shubham-spreadd) [inline src/stories/Alert.js]: isHover, border & outline should be managed by variant

File: src/theme.js [hunk 2]
```diff
@@ -1,42 +1,19 @@
...
+    cardButtonBg: "#E90064",
+    cardButtonHoverBg: "#FF5F9E",
+    textLinkColor: "#E26EE5",
+    iconAfterLinkColor: "#ccc",
+    buttonOutlinedSecondaryBorder: "#E26EE5",
+    buttonOutlinedSecondaryBgHover: "#E26EE5",
+    buttonOutlinedSecondaryColorHover: "#fff",
+    badgeBg: "#FF3D00",
+    badgeText: "#fff",
+    white: "#f6f9fc",
```
Comment (shubham-spreadd) [inline src/theme.js]: remove redundant suffix & unused colors

**Review by shubham-spreadd — CHANGES_REQUESTED** [review]

File: README.TXT
Comment (tarakhant1297) [inline README.TXT:1]: @KBhautik6 remove this empty file

File: src/stories/Alert/Alert.Styled.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  border: 2px solid;
+  width: 100%;
+  padding: 10px 20px;
+  text-align: left;
+  cursor: pointer;
+  text-transform: uppercase;
+  font-size: 1.4rem;
+  ${({ type, isoutline, border }) => css`
+    background-color: ${isoutline ? "transparent" : theme?.[type]};
+    color: ${isoutline ? theme?.[type] : "white"};
```
Comment (tarakhant1297) [inline src/stories/Alert/Alert.Styled.js]: @KBhautik6 read color from theme

File: src/stories/Alert/Alert.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Alert } from "./Alert.Styled";
+
+export const SimpleAlert = ({
```
Comment (tarakhant1297) [inline src/stories/Alert/Alert.js]: @KBhautik6 file name and const should be same

File: src/stories/Alert/Alert.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+  text,
+  variant,
+  isoutline,
+  border,
+  ishover,
+}) => (
+  <Alert
+    type={type}
+    variant={variant}
+    isoutline={isoutline}
```
Comment (tarakhant1297) [inline src/stories/Alert/Alert.js]: @KBhautik6 isoutline rename to `isOutline `

File: src/stories/Alert/Alert.js [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+    border={border}
+    ishover={ishover}
```
Comment (tarakhant1297) [inline src/stories/Alert/Alert.js]: @KBhautik6 we don't need to pass this props we manage using  css

File: src/stories/Alert/Alert.stories.js [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+      text="Warning! This is a Warning Alert - Check it now!"
+    />
+  </StyledHeading>
+);
+
+export const OutlineAlerts = () => (
+  <Styledheading>
+    <SimpleAlert
+      type="primary"
+      variant="outlined"
```
Comment (tarakhant1297) [inline src/stories/Alert/Alert.stories.js:39]: @KBhautik6 why you pass isoutline flag as we pass varinat="outlined"

**Review by tarakhant1297 — CHANGES_REQUESTED** [review]

File: src/stories/Alert/Alert.js (hunk 5 above)
Comment (KBhautik6) [reply src/stories/Alert/Alert.js]: okay

File: src/stories/Alert/Alert.Styled.js [hunk 8]
```diff
@@ -0,0 +1,23 @@
+import styled, { css } from "styled-components";
+import { theme } from "../../theme";
+
+export const Alert = styled.button`
```
Comment (tarakhant1297) [inline src/stories/Alert/Alert.Styled.js]: @KBhautik6 add styled prefix

**Review by tarakhant1297 — APPROVED** [review]

**Pull request merged**